blueprint/blueprint.h

```diff
@@ -13,58 +13,58 @@
     minimumCppStandard: 17
     dependencies:       juce_gui_basics
     searchpaths:        ./duktape/src-noline/ ./yoga
-   END_JUCE_MODULE_DECLARATION
+    END_JUCE_MODULE_DECLARATION
 *******************************************************************************/
 
 #pragma once
 
 #define BLUEPRINT_H_INCLUDED
 
+#include <map>
+#include <unordered_map>
+
 #include <juce_gui_basics/juce_gui_basics.h>
 
 #if JUCE_MODULE_AVAILABLE_juce_audio_processors
     #include <juce_audio_processors/juce_audio_processors.h>
 #endif
 
-#include "yoga/yoga/YGMacros.h"
+//==============================================================================
+#if JUCE_EXCEPTIONS_DISABLED
```

Comment:
@nick-thompson  Here are some checks to be sure your module works as expected and doesn't surprise the user with anything.